File: Cpp/Pc.cpp
class Pc {

    /* Private Attribute */
    private:

        Processor attProcie;
        Disk attDisk;
        Ram attRam;
        int totalPrice;

    /* Public Attribute and Method */
    public:

        /* Constructor */
        Pc() {
            
        }

        Pc(Processor attProcie, Disk attDisk, Ram attRam, int totalPrice) {
            this->attProcie = attProcie;
            this->attDisk = attDisk;
            this->attRam = attRam;
            this->totalPrice = totalPrice;
        }

        /* Setter dan Getter Processor */
        void setProcessor(Processor attProcie) {
            this->attProcie = attProcie;
        }

        Processor getType() {
            return this->attProcie;
        }

        /* Setter dan Getter Disk */
        void setDisk(Disk attDisk) {
            this->attDisk = attDisk;
        }

        Disk getDisk() {
            return this->attDisk;
        }

        /* Setter dan Getter Ram */
        void setRam(Ram attRam) {
            this->attRam = attRam;
        }

        Ram getRam() {
            return this->attRam;
        }

        /* Setter dan Getter Total Price */
        void setTotalPrice() {
            this->totalPrice = this->attDisk.getPrice() + this->attProcie.getPrice() + this->attRam.getPrice();
        }

        int getTotalPrice() {
            return this->totalPrice;
        }

        /* Print data Pc */
        void printPcData() {
            cout << "__My PC Build Plan___________________" << endl;
            cout << "[Processor]" << endl;
            cout << "Name : " << this->attProcie.getName() << endl;
            cout << "Price : RP" << this->attProcie.getPrice() << endl;
            cout << "" << endl;
            cout << "[Disk]" << endl;
            cout << "Type : " << this->attDisk.getType() << endl;
            cout << "Capacity : " << this->attDisk.getCapacity() << endl;
            cout << "Price : RP" << this->attDisk.getPrice() << endl;
            cout << "" << endl;
            cout << "[RAM]" << endl;
            cout << "Capacity : " << this->attRam.getCapacity() << endl;
            cout << "Price : RP" << this->attRam.getPrice() << endl;
            cout << "_____________________________________" << endl;
            cout << "Total Build Price : RP" << this->getTotalPrice() << endl;
        }

        /* Destructor */
        ~Pc() {

        }

};

File: php/Pc.php
<?php

include "Processor.php";
include "Disk.php";
include "Ram.php";

class Pc {

    /* Private Attribute */
    private $attProcie;
    private $attDisk;
    private $attRam;
    private $totalPrice = 0;

    /* Constructor */
    public function __construct() {
        $this->attProcie = new Processor();
        $this->attDisk = new Disk();
        $this->attRam = new Ram();
        $this->totalPrice = 0;
    }

    /* Setter dan Getter Processor */
    public function setProcessor($attProcie) {
        $this->attProcie = $attProcie;
    }

    public function getProcessor() {
        return $this->attProcie;
    }

    /* Setter dan Getter Disk */
    public function setDisk($attDisk) {
        $this->attDisk = $attDisk;
    }

    public function getDisk() {
        return $this->attDisk;
    }

    /* Setter dan Getter Ram */
    public function setRam($attRam) {
        $this->attRam = $attRam;
    }

    public function getRam() {
        return $this->attRam;
    }

    /* Setter dan Getter Total Price */
    public function setTotalPrice() {
        $this->totalPrice = $this->attProcie->getPrice() + $this->attDisk->getPrice() + $this->attRam->getPrice();
    }

    public function getTotalPrice() {
        return $this->totalPrice;
    }

    /* Print data komponen Pc */
    public function printPcData() {
        echo "<br /> __My PC Build Plan___________________";
        echo "<br /> [Processor]";
        echo "<br /> Name : ". $this->attProcie->getName();
        echo "<br /> Price : RP". $this->attProcie->getPrice();
        echo "<br />";
        echo "<br /> [Disk]";
        echo "<br /> Type : ". $this->attDisk->getType();
        echo "<br /> Capacity : ". $this->attDisk->getCapacity();
        echo "<br /> Price : RP". $this->attDisk->getPrice();
        echo "<br />";
        echo "<br /> [Ram]";
        echo "<br /> Capacity : ". $this->attRam->getCapacity();
        echo "<br /> Price : RP". $this->attRam->getPrice();
        echo "<br /> _____________________________________";
        echo "<br /> Price : RP". $this->getTotalPrice();
    }

    /* Destructor */
    public function __destruct() {

	}

}

?>